### PR TITLE
this is the latest one :-)))

### PR DESCRIPTION
**Pull request by dilippoudel**

### CODE REVIEW

File: src/controllers/uploader.ts [hunk 1]
```diff
@@ -12,20 +12,22 @@ aws.config.update({
...
 })
-const S3_BUCKET = process.env.bucket
+const S3_BUCKET = process.env.BUCKET
 export const uploadImages = (
   req: Request,
   res: Response,
   next: NextFunction
 ) => {
   try {
+    console.log('the user is ', req.user)
```
Comment (Wyfy0107) [inline src/controllers/uploader.ts]: 🐞

File: src/controllers/uploader.ts [hunk 2]
```diff
@@ -12,20 +12,22 @@ aws.config.update({
...
     const s3 = new aws.S3()
+    console.log(req.body, '%%%%%%%%%%%')
```
Comment (Wyfy0107) [inline src/controllers/uploader.ts]: 🐛

File: src/routers/uploadProfile.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import express from 'express'
 import { uploadImages } from '../controllers/uploader'
+import tokenVerify from '../middlewares/tokenVerify'
```
Comment (Wyfy0107) [inline src/routers/uploadProfile.ts:3]: Use this in the router

**Review by Wyfy0107 — CHANGES_REQUESTED** [review]

**Review by Wyfy0107 — APPROVED** [review]
💯

**Comment by dilippoudel** [comment]
can i merge it now?

**Pull request merged**